packages/react-native-codegen/src/parsers/__tests__/utils-test.js

```diff
@@ -115,3 +116,107 @@ describe('createParserErrorCapturer', () => {
     });
   });
 });
+
+describe('visit', () => {
```

Comment:
I tried to test each branch, jest gives me 100% coverage but please tell me if you see a way to improve these tests.

Reply:
They looks good to me!